WordPress/src/test/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/AllTimeStatsUseCaseTest.kt

```diff
@@ -153,7 +153,7 @@ class AllTimeStatsUseCaseTest : BaseUnitTest() {
             assertThat(this.endColumn.value).isEqualTo(viewsBestDayTotal.toString())
             assertThat(this.endColumn.highest).isEqualTo(bestDayTransformed)
         }
-        verify(statsWidgetUpdaters, times(2)).updateAllTimeWidget(siteId)
+        verify(statsWidgetUpdaters, times(1)).updateAllTimeWidget(siteId)
```

Comment:
Do you know why this is being called once now?

Reply:
Answering only here, but the same reasoning for all the other ones.

```
                if (!state.cached) {
                    val updatedCachedData = loadCachedData()
                    if (domainModel != updatedCachedData) {
                        domainModel = updatedCachedData
                        updateState()
                    }
```

That was what was removed in `evaluateState`. We were actually calling it twice. Once to check if we have cached data. And we also called it in `evaluateState`. Now it's only called once. Let me know if that makes sense.